app/Http/Controllers/Api/NotesController.php

```diff
@@ -31,4 +31,13 @@ public function store(Request $request)
 
         return response()->json(Helper::formatStandardApiResponse('success'));
     }
+
+    public function update(Request $request)
+    {
```

Comment:
These are stubs, yes?

Reply:
yes, those are not complete. that was all framing for my edification. Sorry for any confusion.

Reply:
No worries - just clarifying :)